assets/js/components/user-input/UserInputSelectOptions.js

```diff
@@ -153,6 +153,7 @@ export default function UserInputSelectOptions( { slug, options, max } ) {
 							disabled={ disabled }
 						/>
 					</TextField>
+					<label htmlFor={ `${ slug }-select-options` } className="screen-reader-text">Enter your own answer here</label>
```

Comment:
@Hazlitte this needs to be translatable.

Reply:
@eugene-manuilov 👍